en/config/autotune.md

```diff
@@ -0,0 +1,223 @@
+5. Select either the *Rate Controller* or *Attitude Controller* tabs.
+   Ensure that the **Autotune enabled** button is enabled (this will display the **Autotune** button and remove the manual tuning selectors).
+6. Stop moving the joysticks and click on the _Autotune_ button (Fig 1).
+   Read the warning popup and click on **OK** to start tuning.
+5. The drone will first start to perform quick roll motions followed by pitch and yaw motions.
+   The progress is shown in the progress bar, next to the _Autotune_ button.
+6. For fixed wing vehicles, by default the tuning will be applied and tested immediately (and reverted if automatic testing detects a problem).
+   For multicopters, by default you will have to land and disarm before the new tuning parameters are applied.
+   Takeoff carefully and manually test that the vehicle is stable.
+7. If any strong oscillations occur, land immediately and follow the instructions in the [Troubleshooting](#troubleshooting) section below.
+
+VTOL vehicles are tuned Tuning is the same as multicopter in MC mode and fixed-wing in FW mode.
+You should repeat the tuning process for each case.
+
+## Parameters
+
+By default MC vehicles land before parameters are applied, while FW vehicles apply the parameters in-air and then test that the controllers work properly.
+This behaviour can be configured using the [MC_AT_APPLY](../advanced_config/parameter_reference.md#MC_AT_APPLY) and [FW_AT_APPLY](../advanced_config/parameter_reference.md#FW_AT_APPLY) parameters respectively:
+
+* `0`: the gains are not applied.
+  This is used for testing purposes if the user wants to inspect results of the auto-tuning algorithm without using them directly.
+* `1`: apply the gains after disarm (default for multirotors).
+  The operator can then test the new tuning while taking-off carefully.
+* `2`: apply immediately (default for fixed-fings).
+  The new tuning is applied, disturbances are sent to the controller and the stability is monitored during the next 4 seconds.
+  If the control loop is unstable, the control gains are immediately reverted back to their previous value.
+  If the test passes, the pilot can then use the new tuning.
+
+
+Fixed wing vehicles (only) can tune which axes are tuned using the [FW_AT_AXES](../advanced_config/parameter_reference.md#FW_AT_AXES) bitmask parameter:
+
+* bit `0`: roll (default)
+* bit `1`: pitch (default)
+* bit `2`: yaw
+
+
+### Troubleshooting
+
+#### The drone oscillates when performing the testing maneuvers prior to the multirotor auto-tuning
+
+* slow oscillations (1 oscillation per second or slower): this often occurs on large platforms and means that the attitude loop is too fast compared to the rate loop -> decrease `MC_[ROLL|PITCH]_P` by steps of 1.0.
+* fast oscillations (more than 1 oscillation per second): this is because the gain of the rate loop is too high -> decrease `MC_[ROLL|PITCH|YAW]RATE_K` by steps of 0.1
```

Comment:
```suggestion
* fast oscillations (more than 1 oscillation per second): this is because the gain of the rate loop is too high.
   - **Multicopter:** decrease `MC_[ROLL|PITCH|YAW]RATE_K` by steps of 1.0
   - **Fixed-wing:** decrease `FW_[RPY]R_P` by steps of 0.1
```